-1 
Tested at: 6987a1abb5eb2b37614fa120d0fd2f35b71175a5
 I found an error when building: 
 
 <pre>gmake[1]: *** [CompilePython] Error 1
gmake[1]: Target `PostBuild' not remade because of errors.
gmake[1]: Leaving directory `/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-05-26-2300'
gmake: *** [src] Error 2
gmake: Target `all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-9276/7864/summary.html

--- a/Validation/RecoEgamma/python/ElectronMcSignalPostValidatorPt1000_cfi.py
+++ b/Validation/RecoEgamma/python/ElectronMcSignalPostValidatorPt1000_cfi.py
@@ -2,7 +2,7 @@
 import FWCore.ParameterSet.Config as cms
 
 electronMcSignalHistosCfg = cms.PSet(
-  EfficiencyFlag = cms.bool(False), StatOverflowFlag = cms.bool(False)
+  EfficiencyFlag = cms.bool(True), StatOverflowFlag = cms.bool(False)
 )
 
 electronMcSignalPostValidatorPt1000 = cms.EDAnalyzer("ElectronMcSignalPostValidator",

--- a/Validation/RecoEgamma/test/ElectronMcSignalHistos.txt
+++ b/Validation/RecoEgamma/test/ElectronMcSignalHistos.txt
@@ -51,6 +51,9 @@ ElectronMcSignalValidator/h_scl_EoEtrue_endcaps_new_deegap  1 1 1 0
 ElectronMcSignalValidator/h_scl_EoEtrueVsrecOfflineVertices_pfx 1 1 0 0
 ElectronMcSignalValidator/h_scl_EoEtrueVsrecOfflineVertices_barrel_pfx 1 1 0 0
 ElectronMcSignalValidator/h_scl_EoEtrueVsrecOfflineVertices_endcaps_pfx 1 1 1 0
+ElectronMcSignalValidator/h_scl_bcl_EtotoEtrue              1 1 0 0
+ElectronMcSignalValidator/h_scl_bcl_EtotoEtrue_barrel       1 1 0 0
+ElectronMcSignalValidator/h_scl_bcl_EtotoEtrue_endcaps      1 1 1 0
 ElectronMcSignalValidator/h_ele_EtaMnEtaTrue                1 1 0 0
 ElectronMcSignalValidator/h_ele_EtaMnEtaTrue_barrel         1 1 0 0
 ElectronMcSignalValidator/h_ele_EtaMnEtaTrue_endcaps        1 1 1 0
@@ -129,9 +132,6 @@ ElectronMcSignalValidator/h_scl_E2x5max_endcaps             1 1 1 0
 ElectronMcSignalValidator/h_scl_E5x5                        1 1 0 0
 ElectronMcSignalValidator/h_scl_E5x5_barrel                 1 1 0 0
 ElectronMcSignalValidator/h_scl_E5x5_endcaps                1 1 1 0
-ElectronMcSignalValidator/h_scl_bcl_EtotoEtrue              1 1 0 0
-ElectronMcSignalValidator/h_scl_bcl_EtotoEtrue_barrel       1 1 0 0
-ElectronMcSignalValidator/h_scl_bcl_EtotoEtrue_endcaps      1 1 1 0
 ElectronMcSignalValidator/h_ele_HoEVsEta_pfx                0 1 1 1
 
 Electron track variables

--- a/Validation/RecoEgamma/test/ElectronMcSignalHistosPt1000.txt
+++ b/Validation/RecoEgamma/test/ElectronMcSignalHistosPt1000.txt
@@ -48,6 +48,9 @@ ElectronMcSignalValidatorPt1000/h_scl_EoEtrue_barrel_new_phigap   1 1 1 0
 ElectronMcSignalValidatorPt1000/h_scl_EoEtrue_ebeegap_new         1 1 1 0
 ElectronMcSignalValidatorPt1000/h_scl_EoEtrue_endcaps_new         1 1 0 0
 ElectronMcSignalValidatorPt1000/h_scl_EoEtrue_endcaps_new_deegap  1 1 1 0
+ElectronMcSignalValidatorPt1000/h_scl_bcl_EtotoEtrue              1 1 0 0
+ElectronMcSignalValidatorPt1000/h_scl_bcl_EtotoEtrue_barrel       1 1 0 0
+ElectronMcSignalValidatorPt1000/h_scl_bcl_EtotoEtrue_endcaps      1 1 1 0
 ElectronMcSignalValidatorPt1000/h_ele_EtaMnEtaTrue                1 1 0 0
 ElectronMcSignalValidatorPt1000/h_ele_EtaMnEtaTrue_barrel         1 1 0 0
 ElectronMcSignalValidatorPt1000/h_ele_EtaMnEtaTrue_endcaps        1 1 1 0
@@ -107,9 +110,9 @@ ElectronMcSignalValidatorPt1000/h_ele_HoE_bc_endcaps              1 1 1 0
 ElectronMcSignalValidatorPt1000/h_ele_hcalDepth1OverEcalBc		    1 1 0 0 
 ElectronMcSignalValidatorPt1000/h_ele_hcalDepth1OverEcalBc_barrel	    1 1 0 0	
 ElectronMcSignalValidatorPt1000/h_ele_hcalDepth1OverEcalBc_endcaps	    1 1 1 0
-ElectronMcSignalValidatorPt1000/h_ele_hcalDepth2OverEcalBc		    1 1 0 0	
+ElectronMcSignalValidatorPt1000/h_ele_hcalDepth2OverEcalBc		        1 1 0 0	
 ElectronMcSignalValidatorPt1000/h_ele_hcalDepth2OverEcalBc_barrel	    1 1 0 0
-ElectronMcSignalValidatorPt1000/h_ele_hcalDepth2OverEcalBc_endcaps        1 1 1 0
+ElectronMcSignalValidatorPt1000/h_ele_hcalDepth2OverEcalBc_endcaps      1 1 1 0
 ElectronMcSignalValidatorPt1000/h_ele_HoE_fiducial                1 1 1 0
 ElectronMcSignalValidatorPt1000/h_scl_sigietaieta                 1 1 0 0
 ElectronMcSignalValidatorPt1000/h_scl_sigietaieta_barrel          1 1 0 0

